.travis.yml

```diff
@@ -20,10 +20,11 @@ addons:
 env:
     global:
         # Set defaults to avoid repeating in most cases
-        - NUMPY_VERSION=1.9
+        - PYTHON_VERSION=3.5
+        - NUMPY_VERSION=1.10
```

Comment:
Should we have ``NUMPY_VERSION=stable`` here to always update to the latest stable version?